### PR TITLE
Add ceph storage gating configuration for openshift/console

### PR DESCRIPTION
**Pull request by a2batic**
Signed-off-by: Kanika Murarka <kmurarka@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml [hunk 1]
```diff
@@ -90,6 +90,86 @@ presubmits:
...
+        - --template=/usr/local/ceph-storage-plugin
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: JOB_NAME_SAFE
+          value: ceph-storage-plugin
+        - name: TEST_COMMAND
+          value: ./test-cypress.sh -p ceph -h true
```
Comment (cloudbehl) [inline ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml:126]:
```suggestion
          value: ./test-prow-e2e.sh ceph
```

Comment (cloudbehl) [reply ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml:126]: https://github.com/openshift/console/pull/7308

**Comment by bipuladh** [comment]
/hold
until https://github.com/openshift/console/pull/7308 merges

**Comment by cloudbehl** [comment]
@a2batic can you rebase the PR and also add a new parameter of instance_type.

https://github.com/openshift/release/pull/14350

File: ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml [hunk 2]
```diff
@@ -98,6 +98,90 @@ presubmits:
...
+        - --report-username=ci
+        - --secret-dir=/usr/local/ceph-storage-plugin-cluster-profile
+        - --target=ceph-storage-plugin
+        - --template=/usr/local/ceph-storage-plugin
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: JOB_NAME_SAFE
```
Comment (cloudbehl) [inline ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml:132]: can you add this?
```
    - name: COMPUTE_NODE_TYPE
      value: m5.2xlarge
```

Comment (bipuladh) [reply ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml:132]: NIT: m4.2xlarge

File: ci-operator/config/openshift/console/openshift-console-master.yaml [hunk 3]
```diff
@@ -90,6 +90,22 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-azure
+- artifact_dir: /tmp/artifacts
+  as: ceph-storage-plugin
+  steps:
+    cluster_profile: azure4
```
Comment (umangachapagain) [inline ci-operator/config/openshift/console/openshift-console-master.yaml:96]:
```suggestion
    cluster_profile: aws
```

File: core-services/template-deprecation/_allowlist.yaml [hunk 4]
```diff
@@ -196,6 +196,9 @@ templates:
         canary-release-openshift-origin-installer-e2e-aws-4.8-cnv:
           generated: false
           kind: unknown
+        pull-ci-openshift-console-master-ceph-storage-plugin:
+          generated: true
+          kind: presubmit
```
Comment (umangachapagain) [inline core-services/template-deprecation/_allowlist.yaml:201]:
```suggestion
```

**Review by umangachapagain — CHANGES_REQUESTED** [review]

**Comment by cloudbehl** [comment]
test pj-rehearse

**Comment by bipuladh** [comment]
/close
Replacement PR: https://github.com/openshift/release/pull/14682

**Pull request closed without merging**